thoth/storages/graph/postgres.py

```diff
@@ -3574,10 +3704,25 @@ def _create_python_package_version(
 
         return python_package_version
 
+    @staticmethod
+    def _create_fuzzy_hash(sorted_ids: List[int]) -> str:
+        """Create fuzzy hash from sorted list of integers.
+
+        Reference: https://pypi.org/project/python-ssdeep/
+        Reference: https://docs.python.org/3/library/stdtypes.html#int.to_bytes
+        """
+        hash_ = ""
+        for object_id in sorted_ids:
+            # lenght of bytes required for the integer
```

Comment:
str(length/lenght)

Reply:
what is it? :D

Reply:
# (lenght -> length) of bytes required for the integer :stuck_out_tongue:

Reply:
thanks sir :D I didn't see the typo actually :rocket: